packages/web-app-user-management/src/views/Users.vue

```diff
@@ -81,24 +83,63 @@ import { clientService } from 'web-pkg/src/services'
 import { useTask } from 'vue-concurrency'
 import { bus } from 'web-pkg/src/instance'
 import { mapActions } from 'vuex'
+import axios from 'axios'
 
 export default {
   components: { UsersList, AppLoadingSpinner, NoContentMessage, CreateUserModal, DeleteUserModal },
   setup() {
     const store = useStore()
     const users = ref([])
+    const roles = ref([])
+    const userAssignments = ref([])
     const graphClient = clientService.graphAuthenticated(
       store.getters.configuration.server,
       store.getters.getToken
     )
 
+    const loadRolesTask = useTask(function* (signal, ref) {
+      const rolesResponse = yield axios.post(
+        '/api/v0/settings/roles-list',
+        {},
+        {
+          headers: {
+            authorization: store.getters.getToken
+          }
+        }
+      )
+      roles.value = rolesResponse.data.bundles
+    })
+
+    const loadUserAssignmentTask = useTask(function* (signal, ref) {
+      const userAssignmentResponse = yield axios.post(
+        '/api/v0/settings/assignments-list',
```

Comment:
Not quite sure about the route,  maybe we need to retrieve the server uri from configuration ?
@kulmann

Reply:
nope, fine to hardcode it. it's not announced anywhere, so you would need to build something first to announce the endpoint. and it will be moved to graph sometime this year.

Reply:
but doing `${this.configuration.server}/api....` might be saver?